tests/integration_tests/config/simulate_prod_gamma_20_deg_North.yml

```diff
@@ -2,7 +2,7 @@ CTA_SIMPIPE:
   APPLICATION: simtools-simulate-prod
   TEST_NAME: gamma_20_deg
   CONFIGURATION:
-    PRODUCTION_CONFIG: ./tests/resources/prod_multi_config_test.yml
+    PRODUCTION_CONFIG: ./tests/resources/prod_multi_config_test_North.yml
     MODEL_VERSION: prod6
     SITE: North
     PRIMARY: gamma
```

Comment:
no integration_tests for north here?

Reply:
There is one test implemented, but in the other PR: https://github.com/gammasim/simtools/pull/875